third_party/ienn/src/ie_compilation.cpp

```diff
@@ -350,7 +352,8 @@ int32_t Compilation::AddConvolution(const Operation& operation) {
     }
     // add bias and fused-code
     const uint32_t output_index = operation.outputs[0];
-    AddBiasWithBroadcasting(bias_index, output_index);
+    SizeVector bias_dims{1, params.bias_length, 1, 1};
```

Comment:
How about adding comments to note that we need to reshape bias to make NCHW layout for resolving mixed layout error just for clDNN?